modules/seqexec/web/client/src/main/resources/dev.js

```diff
@@ -6,7 +6,7 @@ import React from "react";
 if (process.env.NODE_ENV !== "production") {
   const { whyDidYouUpdate } = require("why-did-you-update");
   whyDidYouUpdate(React, {
-    exclude: ["Draggable", "DraggableCore", "AutoSizer", "DiodeWrapper"]
+    exclude: ["Draggable", "DraggableCore", "AutoSizer", "DiodeWrapper", "Login"]
```

Comment:
How is `Login` a false positive. It seems it should be possible to make it reusable

Reply:
I'm calling false positives the cases where the plugin tells the rerender is not necessary but it is. This is happening with `Login` and other components.